expo/gbe/views.py

```diff
@@ -1417,8 +1431,11 @@ def review_volunteer(request, volunteer_id):
         return view_volunteer(request, volunteer_id)
     conference, old_bid = get_conf(volunteer)
     volunteer_prof = volunteer.profile
-    volform = VolunteerBidForm(instance=volunteer,
-                               prefix='The Volunteer')
+    volform = VolunteerBidForm(
+        instance=volunteer,
+        prefix='The Volunteer', 
+        available_windows=volunteer.conference.windows(),
+        unavailable_windows=volunteer.conference.windows())
```

Comment:
Also fixed. (styling isn't nice, but I'm not throwing any more time away on this part)